requirements/base.txt

```diff
@@ -4,17 +4,16 @@
 #
 #    make upgrade
 #
-
 bleach==2.1.4
 boto==2.39.0
-certifi==2019.9.11        # via requests
+certifi==2019.11.28       # via requests
 chardet==3.0.4            # via requests
 defusedxml==0.5.0
 django-model-utils==3.0.0
 django==1.11.26
 git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework
 edx-i18n-tools==0.4.8
-edx-submissions==3.0.2
+edx-submissions==3.0.3
```

Comment:
Just a reminder that you'll have to make sure edx-submissions gets upgraded to 3.0.3 in edx-platform, too, when you upgrade ora2 there.